djstripe/settings.py

```diff
@@ -54,6 +54,17 @@ def get_callback_function(setting_name, default=None):
 subscriber_request_callback = get_callback_function("DJSTRIPE_SUBSCRIBER_MODEL_REQUEST_CALLBACK",
                                                     default=(lambda request: request.user))
 
+
+def _get_idempotency_key(object_type, action, livemode):
+    from .models import IdempotencyKey
+    action = "%s:%s" % (object_type, action)
```

Comment:
"{}:{}".format(object_type, action). I can't wait until we can drop py2 and jump on f strings